loader_hub/file/flat_pdf/base.py

```diff
@@ -0,0 +1,80 @@
+"""Simple reader that reads flatten PDFs."""
+import os
+import pathlib
+import warnings
+from pathlib import Path
+
+from llama_index.readers.base import BaseReader
+from llama_index.readers.schema.base import Document
+
+
+class FlatPdfReader(BaseReader):
+
+    pdf_dir: Path
```

Comment:
won't this error since it's a class attribute? it's not shown in your README how to specify these args (`pdf_dir`, `pdf_name`, `pdf_pages_count`, etc.)

Reply:
They were only being used internally, a "global variable" logic, but with your comment, I looked again and with that logic it may cause strange behavior as the `FlatPdfReader` object can be instantiated once and used for many different pdfs , so I made the correction for these attributes to become local scope variables in the methods.